pymc3/distributions/continuous.py

```diff
@@ -298,7 +298,7 @@ def __init__(self, sd=None, tau=None, *args, **kwargs):
         assert_negative_support(sd, 'sd', 'HalfNormal')
 
     def random(self, point=None, size=None, repeat=None):
-        sd = draw_values([self.sd], point=point)
+        sd = draw_values([self.sd], point=point)[0]
```

Comment:
is it effect the speed? Since you commented in #2296 that `draw_values` might also cause slowdown in `random` method.